pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -196,6 +196,14 @@ message LinuxSandboxSecurityContext {
     bool privileged = 6;
 }
 
+// Sysctls holds linux sysctls config for the sandbox.
+message Sysctls {
+    // List of safe sysctls which are set for the sandbox.
+    map<string, string> sysctls = 1;
+    // List of unsafe sysctls which are set for the sandbox.
+    map<string, string> unsafe_sysctls = 2;
```

Comment:
I'm not sure whether runtimes want to distinguish those two. 

@yujuhong @mrunalp What do you think?